developer/src/tike/xml/help/contexthelp.xml

```diff
@@ -387,6 +387,155 @@
+    </Control>
+
+    <Control Name="editIndentSize" Title="Indent size">
+      <p>The indent size, measured in number of spaces.</p>
+    </Control>
+
+    <Control Name="chkLinkFontSizes" Title="Link quoted font size to primary font size">
+      <p>If checked, the quoted font size will be the same as the default font size.</p>
+    </Control>
+
+    <Control Name="cmdDefaultFont" Title="Default font">
+      <p>Sets the font for ordinary text in the editor.</p>
+    </Control>
+
+    <Control Name="cmdQuotedFont" Title="Quoted font">
+      <p>Sets the font for displaying comments text and strings in Keyman source files in single or double quotes.</p>
+    </Control>
+
+    <Control Name="cbEditorTheme" Title="Editor theme">
+      <p>Sets the display theme and syntax highlighting options for the editor. A custom theme can be defined in a JSON file.</p>
+    </Control>
+
+    <Control Name="chkUseOldDebugger" Title="Enable test window (from version 5.0)">
+      <p>Enables the Keyman 5-style Test window instead of the debugger.</p>
+    </Control>
+
+    <Control Name="chkDebuggerBreakWhenExitingLine" Title="Breakpoints fire also when exiting line">
+      <p>Breakpoints will fire when a rule has finished being processed also.</p>
+    </Control>
+
+    <Control Name="chkDebuggerSingleStepAfterBreak" Title="Turn on single step after breakpoint">
+      <p>Activates single-step mode after a breakpoint fires.</p>
+    </Control>
+
+    <Control Name="chkDebuggerShowStoreOffset" Title="Show matched character offsets in stores">
+      <p>Displays numeric indices of characters in stores</p>
+    </Control>
+
+    <Control Name="chkDebuggerAutoRecompile" Title="Automatically recompile if no debug information available">
+      <p>When the debugger starts, rebuild a keyboard without prompting if the keyboard 
+      has no debug symbols included.</p>
+    </Control>
+
+    <Control Name="chkDebuggerAutoResetBeforeCompiling" Title="Automatically reset debugger before recompiling">
+      <p>update is coming</p>
+    </Control>
+
+    <Control Name="editWebHostDefaultPort" Title="Debugger tab">
+      <p></p>
+    </Control>
+
+    <Control Name="chkCharMapAutoLookup" Title="Find character under cursor automatically">
+      <p>The character beneath the cursor in the edit window will be highlighted in the 
+      Character Map. This will intelligently parse the character data under the cursor so that 
+      a character code (e.g. U+1234) will be highlighted correctly.</p>
+    </Control>
+
+    <Control Name="cmdCharMapRebuildDatabase" Title="Update database">
+      <p>Rebuilds the Unicode character database from source unicodedata.txt and blocks.txt. 
+      These files can be downloaded from the Unicode website at http://www.unicode.org/ucd/. 
+      This lets you update the character map with a newer version of Unicode. 
+      Keyman Developer 10.0 was released with Unicode version 10.0 data.</p>
```

Comment:
This line is very old :-) Is it also old on help.keyman.com?

Reply:
Yes, I got it from [developer 17.0](https://help.keyman.com/developer/17.0/context/options#:~:text=in%20the%20database.-,Update%20database,-Rebuilds%20the%20Unicode).

Reply:
Let's update that to Keyman Developer 17.0 was released with Unicode version 15.1.

Reply:
Noted!

Reply:
This change is still waiting